network/model.go

```diff
@@ -8,6 +8,116 @@ type CreateInteractionRequest struct {
 	NPCType       model.PlayerID `json:"npc_type,omitempty"`
 }
 
+// TODO figure out the minimum info the client will need
+type GetGameResponse struct {
+	ID              model.GameID                         `json:"id"`
+	Players         []Player                             `json:"players"`
+	PlayerColors    map[model.PlayerID]model.PlayerColor `json:"player_colors,omitempty"`
+	CurrentScores   map[model.PlayerColor]int            `json:"current_scores"`
+	LagScores       map[model.PlayerColor]int            `json:"lag_scores"`
+	Phase           model.Phase                          `json:"phase"`
```

Comment:
How do we want to communicate this to the client? IIRC it's a typedefed int today; perhaps we should send a string? The client is going to need to know how to switch off of it...

actually, do we want to consider this for `model.PlayerColor` too -- sending `green`, `blue`, `red` instead of `0`, `1`, `2`?

Reply:
Yep, I’m thinking it’ll make sense to define `network` models for these things.